posts/2023-06-14-running-spring-boot-3.adoc

```diff
@@ -0,0 +1,163 @@
+blog_description: In version 23.0.0.3 and later, you can easily run a Spring Boot 3.x application from a WAR file on Open Liberty. This post uses the Spring Boot Petclinic sample application and the Liberty Maven plugin to create and run a WAR-based Spring Boot 3.1 application on Open Liberty. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Running a Spring Boot 3.x application WAR file on Liberty
+Chuck Bridgham <https://github.com/cbridgha>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+In version 23.0.0.3 and later, you can easily run a Spring Boot 3.x application from a WAR file on Open Liberty. This post uses the Spring Boot Petclinic sample application and the Liberty Maven plugin to create and run a WAR-based Spring Boot 3.1 application on Open Liberty.
+
+== Prerequisites
+
+- Java 17
+- Open Liberty version 23.0.0.3 or later, with the link:{url-prefix}/docs/latest/reference/feature/servlet-6.0.html[Jakarta Servlet 6.0] (`servlet-6.0`) feature enabled in the `server.xml` file
+
+
+== Getting started with the Spring Petclinic sample application
+
+This tutorial uses the 3.1 release of the link:https://github.com/spring-projects/spring-petclinic[Spring Petclinic sample application].
+
+To run this application on Open Liberty, you can add the link:https://github.com/OpenLiberty/ci.maven[Liberty Maven plugin] to the existing Petclinic `pom.xml` file. You can also configure a package that runs the required `servlet-6.0` feature and other helpful features by default.
+
+1. Clone the sample repository to your machine.
++
+[source,sh]
+----
+git clone https://github.com/spring-projects/spring-petclinic.git 
+cd spring-petclinic
+----
+
+2. Edit the Petclinic `pom.xml` file to build the sample application from a WAR file by using the Liberty Maven plugin.
++
+First, change the name of the sample from `petclinic` to `liberty-spring-petclinic` and add the `war` packaging type:
++
+[source,xml]
+----
+<name>liberty-spring-petclinic</name>
+<packaging>war</packaging>
+----
++
+Next, specify a WAR file name under the `build` section:
++
+[source,xml]
+----
+<build>
+    <finalName>liberty-${project.artifactId}</finalName>
+----
++
+And finally, under the `<build> <plugins>` section, add the Liberty Maven plugin:
++
+[source,xml]
+----
+<plugin>
+  <groupId>io.openliberty.tools</groupId>
+  <artifactId>liberty-maven-plugin</artifactId>
+  <version>3.8.1</version>
+  <configuration>
+      <runtimeArtifact>
+          <groupId>io.openliberty</groupId>
+          <artifactId>openliberty-webProfile10</artifactId>
+          <version>23.0.0.5</version>
+          <type>zip</type>
+      </runtimeArtifact>
```

Comment:
Add this line after </runtimeArtifact>
<looseApplication>false</looseApplication>